lib/assets/stylesheets/_core.scss

```diff
@@ -52,17 +73,17 @@ html, body, div, h1, h2, h3, h4, h5, h6, article, aside, footer, header, hgroup,
 }
 
 a:link {
-  color: $link-colour;
+  color: govuk-colour("blue");
```

Comment:
Suggest enabling global link and paragraph styling in Frontend and removing these link rules.

Reply:
If not, this should be $govuk-link-colour

Reply:
I'm going to raise all of your suggestions as GH suggestions